src/overlays/actors/ovl_En_Gakufu/z_en_gakufu.c

```diff
@@ -5,23 +5,28 @@
  */
 
 #include "z_en_gakufu.h"
+#include "interface/parameter_static/parameter_static.h"
 
 #define FLAGS 0x02000010
 
 #define THIS ((EnGakufu*)thisx)
 
+extern OcarinaSongButtons gOcarinaSongButtons[24];
```

Comment:
Isn't this in variables.h already? Also, I don't know if we'd put externs above or below the other stuff.